### PR TITLE
Replace tabs with spaces

### LINKED ISSUE #81: Convert tabs to spaces

### PR DESCRIPTION
**Pull request by waffle-stomper**
Closes #81 

I was going to submit this to the Devoted fork, but its last commit was four months ago.

### CODE REVIEW

**Comment by Maxopoly** [comment]
@ProgrammerDan should this be merged?

@waffle-stomper this has conflicts after merging your other PR

**Comment by ProgrammerDan** [comment]
Wait, just re-read the title. If this converts _tabs_ to _spaces_ then absolutely not.

We use tabs here for indentation. So if it converts everything to spaces, kill it.

**Comment by ProgrammerDan** [comment]
Yeah no. 

@waffle-stomper please leverage https://github.com/Civcraft/style-guide which is based on a lengthy collaborative agreement between the lead devs.

tl;dr 120 char line lengths, tabs _not_ spaces, and a host of other little things.

**Comment by waffle-stomper** [comment]
@ProgrammerDan - I misread a comment of yours about tabs vs. spaces in another PR and thought spaces were the standard. Thanks for clearing it up.
